rag/main.py

```
@@ -1047,8 +1048,9 @@ def _run_es_review(
   "star_score": <1-10の整数: Situation/Task/Action/Resultの構造が揃っているか>,
   {company_fit_key},
   "length_balance_score": <1-10の整数: 文字数・各要素のバランスが適切か>,
-  "feedback": "<具体性・STAR準拠・企業適合性・文字数について200字以内でアドバイス>",
-  "improved_text": "<元の文章を改善したバージョン（元の文字数の110〜130%を目安）>"
+  "feedback": "<具体性・STAR準拠・企業適合性・文字数について400字程度でアドバイス。企業名が指定されている場合は企業特化アドバイスを含めてください>",
+  "improved_text": "<元の文章を改善したバージョン（元の文字数の110〜130%を目安）>",
```

Ruff (0.15.17)

[warning] String contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF001)